### PR TITLE
2950 custom sensitive text area widget

### LINKED ISSUE #2950: Write a custom component for SSHKeypair objects in UI
Write a custom component that will add `SensitiveText` widget to the UISchema for the private key of SSH keypair list

# Tasks
* [ ] Write a custom `SSHkeyPairComponent` that will display two textarea wdigets for `public_key` and `private_key`
    * [ ] `private_key` needs to be of type `SensitiveText`
    * [ ] Note: Verify that it works with a ssh keypair used from the UI

### PR DESCRIPTION
**Pull request by ilija-lazoroski**
# What does this PR do?

Fixes #2950.
Things left to do:
* [x] Test it with actual SHHKeypair

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Is the TravisCI build passing?
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [x] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by exploiting sshkey-12 with only sshkey pair
* [x] If applicable, add screenshots or log transcripts of the feature working

image
image
image

### CODE REVIEW

**Review by VakarisZ — CHANGES_REQUESTED** [review]
Let's not add more class components. Refactor monkey/monkey_island/cc/ui/src/components/ui-components/SensitiveTextareaInput.js to hooks

**Review by cakekoa — COMMENTED** [review]
Interesting solution -- I wonder how other password fields are implemented

File: CHANGELOG.md [hunk 1]
```diff
@@ -183,6 +183,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Update MongoDB version to 4.4.x. #1924
 - The `/api/telemetry` endpoint allowed arbitrary queries to be submitted,
   which could result in javascript execution. #2503
+- Fixed plaintext private key in SSHKey pair list in UI. #2950
```
Comment (mssalvatore) [inline CHANGELOG.md:186]: This should NOT be in the v2.0.0 changelog.

**Comment by mssalvatore** [comment]
Can we name this better?

image

**Review by mssalvatore — CHANGES_REQUESTED** [review]

**Comment by ilija-lazoroski** [comment]
> Can we name this better?
> 
> image

We can name it the same for every item there...

image

**Review by cakekoa — COMMENTED** [review]
Is there a way we can reduce code duplication between the two SensitiveText components?

**Comment by mssalvatore** [comment]
@ilija-lazoroski 
> We can name it the same for every item there...
We can't name it "Key Pair 1", "Key Pair 2", ...?

**Comment by ilija-lazoroski** [comment]
> @ilija-lazoroski 
> > We can name it the same for every item there...
> 
> We can't name it "Key Pair 1", "Key Pair 2", ...?

I didn't found any way to do it like that. Do we want to spend some time?

**Comment by mssalvatore** [comment]
> Do we want to spend some time?

Nah. It's fine.

**Review by cakekoa — APPROVED** [review]

**Review by shreyamalviya — APPROVED** [review]

**Pull request merged**